pkg/app/server/githubstore/store.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package githubstore
+
+import (
+	"context"
+
+	"github.com/google/go-github/v29/github"
```

Comment:
You can use more latest version like below.
`import "github.com/google/go-github/v45/github"`

Reply:
another place use this version too, so lets update that by a separated PR 👍

Reply:
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/oauth/github/github.go#L23